atom/browser/api/atom_api_web_contents.h

```diff
@@ -425,7 +424,8 @@ class WebContents : public mate::TrackableObject<WebContents>,
 
 #if BUILDFLAG(ENABLE_OSR)
   OffScreenWebContentsView* GetOffScreenWebContentsView() const;
-  OffScreenRenderWidgetHostView* GetOffScreenRenderWidgetHostView() const;
+  OffScreenRenderWidgetHostView* GetOffScreenRenderWidgetHostView()
```

Comment:
I had to keep this `virtual` in `CommonWebContentsDelegate`, since implementing it just there didn't work (basically it comes down to deciding whether we are an OSR enabled `WebContents` or not, and I got a crash even if I tried to use a `dynamic_cast` on the `RenderWidgetHostView`)

I'm open to other options, but I'm out of ideas for now